### DISCUSSION TITLE
Where is the executable located

### DISCUSSION THREAD

**Discussion by thetechnodino** [pulsar-edit/.github#128] [Core Application & Packages]
I do not write code for a living. Learning web development on line. Download Pulsar.  Extract Pulsar.  Look around to find the executable to install Pulsar.

I cannot find the executable to install Pulsar.  Where is the executable located?  

Thanks

John

**Comment by meadowsys** [comment]
The binary downloads should be available here: <https://pulsar-edit.dev/download.html#releases>. From what you mentioned needing to extract, I presume you are either on macos (download zip) or linux (tar.gz). For macos, the app should be the only file in the zip, and you should be able to run that command (`xattr -cr /Applications/Pulsar.app`) and run as is. For linux, maybe the appimage file would be easier? since all you need to do is download it and click on it. If this isn’t helpful, may I ask what OS you’re running and which download you have used so me or someone else can help you better?

**Comment by thetechnodino** [comment]
Windows 11. I overlooked the downloadable because it was in beta. I 
figured there was a "released" version.

---
John Payne

630-815-2989 | www.pde-usa.net [3]

Sola Deo Gloria

On 2023-02-06 18:29, Meadowsys wrote:

> The binary downloads should be available here: 
> https://pulsar-edit.dev/download.html#releases. From what you mentioned 
> needing to extract, I presume you are either on macos (download zip) or 
> linux (tar.gz). For macos, the app should be the only file in the zip, 
> and you should be able to run that command and run as is. For linux, 
> maybe the appimage file would be easier? since all you need to do is 
> download it and click on it. If this isn't helpful, may I ask what OS 
> you're running and which download you have used so me or someone else 
> can help you better?
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/orgs/pulsar-edit/discussions/128#discussioncomment-4887789
[2] 
https://github.com/notifications/unsubscribe-auth/ALM5AVLF7DYVN3NSRPLA5ALWWGCPPANCNFSM6AAAAAAUTG255M
[3] http://www.pde-usa.net

**Reply by Daeraxa** [reply]
The reason it is marked Beta is really because there are just a few aspects related to branding and some backend stuff that needs to be worked out as well as a bunch of goals for upgrades. The beta version (and even the "rolling" cirrus binaries) can be considered full releases and are just as functional as Atom was.

**Comment by thetechnodino** [comment]
Thanks for explaining. Makes sense.

---
John Payne

630-815-2989 | www.pde-usa.net [3]

Sola Deo Gloria

On 2023-02-06 19:09, Daeraxa wrote:

> The reason it is marked Beta is really because there are just a few 
> aspects related to branding and some backend stuff that needs to be 
> worked out as well as a bunch of goals for upgrades. The beta version 
> (and even the "rolling" cirrus binaries) can be considered full 
> releases and are just as functional as Atom was.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/orgs/pulsar-edit/discussions/128#discussioncomment-4888014
[2] 
https://github.com/notifications/unsubscribe-auth/ALM5AVKCA6J4ZA72JIWV6ZLWWGHFNANCNFSM6AAAAAAUTG255M
[3] http://www.pde-usa.net
